src/main/scala/de/upb/cs/uc4/hyperledger/connections/traits/ConnectionExamResultTrait.scala

```diff
@@ -0,0 +1,56 @@
+package de.upb.cs.uc4.hyperledger.connections.traits
+
+import de.upb.cs.uc4.hyperledger.exceptions.traits.{ HyperledgerExceptionTrait, TransactionExceptionTrait }
+
+trait ConnectionExamResultTrait extends ConnectionTrait {
+  final override val contractName: String = "UC4.Exam"
```

Comment:
```suggestion
  final override val contractName: String = "UC4.ExamResult"
```

Currently both Exam and ExamResult Traits have the contract name "exam"